local/docker-compose.yml

```diff
@@ -19,3 +19,8 @@ services:
       MINIO_ACCESS_KEY: "FAKE_ACCESS_KEY"
       MINIO_SECRET_KEY: "FAKE_SECRET_ACCESS_KEY"
     command: "server /data"
+  mock-api:
```

Comment:
You also need to expose the port here otherwise it won't work. To do so, simply add:
```
    ports:
      - "4000:4000"
```

Reply:
correct.